docker/Dockerfile

```diff
@@ -9,14 +14,34 @@ COPY . .
 RUN ./scripts/all.bash
 
 
-FROM ubuntu:22.04
-
-RUN apt update && apt install -y \
-  dnsmasq iproute2 isc-dhcp-client \
-  libpcap-dev ntfs-3g openssh-client \
-  openvswitch-switch qemu-kvm qemu-utils \
+# -- minimega image --
+FROM ${BASE_IMAGE}
+
+# General image metadata
+# Reference: https://github.com/opencontainers/image-spec/blob/main/annotations.md
+ARG BUILD_TIME
+ARG MM_VERSION
+ARG MM_COMMIT
+LABEL org.opencontainers.image.created="${BUILD_TIME}" \
+  org.opencontainers.image.authors="minimega dev team <minimega-dev@sandia.gov>" \
+  org.opencontainers.image.documentation="https://www.sandia.gov/minimega/" \
+  org.opencontainers.image.source="https://github.com/sandia-minimega/minimega" \
+  org.opencontainers.image.version="${MM_VERSION}" \
+  org.opencontainers.image.revision="${MM_COMMIT}" \
+  org.opencontainers.image.vendor="Sandia National Laboratories" \
+  org.opencontainers.image.licenses="GPL-3.0-only" \
+  org.opencontainers.image.title="minimega" \
+  org.opencontainers.image.description="minimega is a tool for launching and managing virtual machines"
+
+ENV DEBIAN_FRONTEND="noninteractive"
```

Comment:
This line is a duplicate, so should be good to remove

Reply:
Line 7 runs in context of gobuilder image build (line 5, `FROM golang:1.18.2 AS gobuilder`), while line 36 is in context of the minimega image build (which starts at line 18).

Reply:
Ah gotcha. Wasn't familiar with multi-stage builds, but that makes sense